Jenkinsfile

```diff
@@ -2,106 +2,68 @@ pipeline {
     agent {label 'mp-vm'}
 
     stages {
-        stage('Step 1: Docker Build Image') {
+        stage('Step 1: Download Git repo') {
```

Comment:
Is this step required? I believe Jenkins does it automatically, otherwise it can't see Jenkinsfile.
Then all "github_local_branch" parameters aren't needed.

Reply:
this is done